daos-stack/daos
# DAOS-12950 test: auto-determine test tags

Skip-test: true
Skip-build: true

- Add a tool to:
  - automatically determine test tags based on files modified.
  - run a lint check on ftest tags
- Add a GHA job for runnings ftest tag lint
- Update Jenkinsfile Get Commit Message stage to inject Test-tag from the new utility

Required-githooks: true

### Before requesting gatekeeper:


### Gatekeeper:
